universe/marathon.json.mustache

```diff
@@ -250,24 +236,19 @@
     "MESOS_MODULES": "{\"libraries\":[{\"file\":\"\/opt\/mesosphere\/libmesos-bundle\/lib\/mesos\/libdcos_security.so\",\"modules\":[{\"name\": \"com_mesosphere_dcos_ClassicRPCAuthenticatee\",\"parameters\":[{\"key\":\"service_account_credential\",\"value\":\"file://secrets/service-account.json\"}]},{\"name\":\"com_mesosphere_dcos_http_Authenticatee\",\"parameters\":[{\"key\":\"service_account_credential\",\"value\":\"file://secrets/service-account.json\"},{\"key\":\"jwt_exp_timeout\",\"value\":\"5mins\"},{\"key\":\"preemptive_refresh_duration\",\"value\":\"30mins\"}]}]}]}",
     {{/service.service_account_secret}}
 
-    {{#storage.local_persistence.enabled}}
-    "CONDA_ENVS_PATH": "{{advanced.sandbox}}/{{storage.local_persistence.volume_path}}/conda/envs:/opt/conda/envs",
-    "CONDA_PKGS_DIRS": "{{advanced.sandbox}}/{{storage.local_persistence.volume_path}}/conda/pkgs:/opt/conda/pkgs",
-    "DCOS_DIR": "{{advanced.sandbox}}/{{storage.local_persistence.volume_path}}/.dcos",
-    "HADOOP_CONF_DIR": "{{advanced.sandbox}}/{{storage.local_persistence.volume_path}}",
-    "HOME": "{{advanced.sandbox}}/{{storage.local_persistence.volume_path}}",
-    "JUPYTER_CONFIG_DIR": "{{advanced.sandbox}}/{{storage.local_persistence.volume_path}}/.jupyter",
-    "JUPYTER_RUNTIME_DIR": "{{advanced.sandbox}}/{{storage.local_persistence.volume_path}}/.local/share/jupyter/runtime",
-    {{/storage.local_persistence.enabled}}
-    {{^storage.local_persistence.enabled}}
+    "CONDA_ENVS_PATH": "{{advanced.sandbox}}/conda/envs:/opt/conda/envs",
```

Comment:
Should we change all of these paths to - `"{{advanced.sandbox}}/data/../.."`?
Like they had done it for local persistence? If advanced.sandbox is not used anywhere else other than these configs then we can change it's default value to `/mnt/mesos/sandbox/data`